data-esnext.js

```diff
@@ -3881,6 +3881,65 @@ exports.tests = [
     },
   ]
 },
+{
+  name: 'Logical Assignment',
+  category: STAGE3,
+  significance: 'small',
+  spec: 'https://github.com/tc39/proposal-logical-assignment/',
+  subtests: [
+    {
+      name: '||=',
+      exec: function () {/*
+        let a;
+        let b = 0;
+        let c = 1;
+        a ||= 2;
+        b ||= 2;
+        c ||= 2;
+        return (a == 2) && (b == 2) && (c == 1);
+      */},
+      res: {
+        firefox76: false,
+        firefox77: firefox.nightly,
+        chrome80: false,
```

Comment:
Can confirm both `safari13` and `safaritp` does not support it either.

Reply:
added safari13 results, safaritp is inherited.